todo-frontend-react-typescript/src/App.tsx

```diff
@@ -83,14 +89,19 @@ class App extends React.Component<{}, { todos: TodoData }> {
               }
           });
 
-     //Using catch & release mechanism
+    // Using catch & release mechanism
     // featureHubRepository
     //   .addPostLoadNewFeatureStateAvailableListener((_) =>
     //                                                this.setState(
     //                                                  {todos: this.state.todos.changeFeaturesUpdated(true)}) );
 
     // featureHubRepository.catchAndReleaseMode = true; // catch feature updates and release later
 
+    featureHubRepository.config.userKey('auntie')
```

Comment:
wonder if we should keep it on the branch for now as the frontend is not ready

Reply:
I made a comment on this in the main README that people should download using the tag as master tracks master and may have unreleased functionality. However, I don't mind keeping it on a branch.